src/inputs/TextField/TextField.tsx

```diff
@@ -0,0 +1,98 @@
+import { useState } from "react";
+import * as React from "react";
+import { Error, Cancel, MagGlass } from "../../icons";
+import classNames from "classnames/bind";
+
+import * as styles from "./TextField.module.css";
+
+const cx = classNames.bind(styles);
+
+interface TextFieldProps {
+  type?: "text" | "password";
+  autoFocus?: boolean;
+  disabled?: boolean;
+  value: string;
+  label?: string;
+  name: string;
+  placeholder?: string;
+  error?: string;
+  hint?: string;
+  id: string;
+  onChange(checked: boolean): void;
```

Comment:
**issue:** It should be `onChange(event: React.ChangeEvent<HTMLInputElement>): void`